src/bp-core/bp-core-rewrites.php

```diff
@@ -294,7 +296,7 @@ function bp_rewrites_get_url( $args = array() ) {
 	 *      @type array $single_item_action_variables The list of BuddyPress single item's action variable URL chunks. Defaults [].
 	 * }
```

Comment:
```suggestion
	 * }
	 * @param array  $args Original arguments used with the function.
```

Reply:
@imath fine with me, did you want a new pull request or will you do it when applying patch in SVN?

Reply:
Awesome, thanks a lot for your feedback, I'll apply it when committing to SVN 👌.